server/src/main/java/org/mixit/cesar/cfp/model/ProposalVote.java

```diff
@@ -0,0 +1,77 @@
+package org.mixit.cesar.cfp.model;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import org.mixit.cesar.site.model.FlatView;
+import org.mixit.cesar.site.model.member.Staff;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(uniqueConstraints={@UniqueConstraint(columnNames={"proposal_id", "voter_id"})})
+public class ProposalVote {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @JsonView(FlatView.class)
+    private Long id;
+
```

Comment:
I think it would be nice to add a comment when we vote for a session. That's why I think it's better to add the buttons in detail screen and not in the list. What do you think about that ?